nixos/modules/services/torrent/transmission.nix

```diff
@@ -222,10 +219,19 @@ in
         };
       };
 
-      package = mkPackageOption pkgs "transmission" {
-        default = "transmission_3";
-        example = "pkgs.transmission_4";
-      };
+      package =
+        mkPackageOption pkgs "transmission" {
+          default = "transmission_4";
+          example = "pkgs.transmission_4";
+        }
+        // {
+          defaultText = ''
+            if lib.versionAtLeast config.system.stateVersion "25.11" then
+              pkgs.transmission_4
+            else
+              «error message»
+          '';
```

Comment:
I'm not sure we need this additional `defaultText`, I'd just stick with the default.